server/controllers/metricsController.js

```diff
@@ -0,0 +1,42 @@
+const { spawn } = require('child_process');
+const fetch = require('node-fetch');
+
+const metricsController = {};
+
+const PROM_URL = 'http://127.0.0.1:9090/api/v1/';
```

Comment:
Is it okay for this URL to be exposed?

Reply:
I assume so as it's really just localhost and not a public facing IP. It should only be accessible through the computer it's being ran on.

Reply:
Is this a URL string that will work for all devs spinning up our app, or is it somehow specific?

Reply:
It should be the default prometheus server port. Can add a where users fill out a form stating the port and ip of their prometheus server in the event they change it